design/one-pager-definition-of-done.md

```diff
@@ -1,54 +0,0 @@
-# Definition of done
```

Comment:
I think it is fine to remove here, but just wondering what our policy is when it comes to retiring design docs? I know we have previously kept some around and just moved to `Status: Defunct`

Ref: https://github.com/crossplane/crossplane/blame/master/design/design-doc-reconciler-patterns.md#L4

Reply:
I think that's a good practice, but this is a little bit of an unusual design doc. It captures process rather than any technical history of the project. I feel okay about removing this one as a special case, given the content has moved into a doc that addresses our process.